core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -64,8 +68,18 @@ object Resource extends ConfigFormat {
     def installTo(target: Path): Unit
 
     def rootEntries: List[String] = {
-      val fs = FileSystems.newFileSystem(path, this.getClass.getClassLoader)
-      Files.list(fs.getPath("/")).iterator.asScala.map(_.toString).toList
+      Try(FileSystems.newFileSystem(path, this.getClass.getClassLoader)) match {
+        case Success(fs) =>
+          Files.list(fs.getPath("/")).iterator.asScala.map(_.toString).toList
+        case Failure(_: ProviderNotFoundException) =>
+          throw new IllegalArgumentException(
+            s"""
+               |The path provided as the plugin path: $path is not valid.
```

Comment:
That would save us quite a while of debug in git-machete-intellij-plugin if it was here earlier 😅